Mathlib/Tactic/Positivity/Basic.lean

```diff
@@ -420,6 +421,44 @@ def evalAscFactorial : PositivityExt where eval {u α} _ _ e := do
     pure (.positive q(Nat.ascFactorial_pos $n $k))
   | _, _, _ => throwError "failed to match Nat.ascFactorial"
 
+section NNRat
+open NNRat
+
+private alias ⟨_, NNRat.num_pos_of_pos⟩ := num_pos
+private alias ⟨_, NNRat.num_ne_zero_of_ne_zero⟩ := num_ne_zero
```

Comment:
Do you think these should just be public (after checking they end up in the right namespace)?

bors d+

Reply:
My stance on whether one sided implications of unconditional iff lemmas without potential for dot notation should exist is "not really".

Btw the reason I added the `NNRat` namespace, as opposed to the
```
open Rat

private alias ⟨_, num_pos_of_pos⟩ := num_pos
private alias ⟨_, num_ne_zero_of_ne_zero⟩ := num_ne_zero
```
just below, is that both pairs of aliases name-conflict! The lemmas aren't autogenerated to guarantee uniqueness.